web/src/lib/components/admin-page/delete-confirm-dialogue.svelte

```diff
@@ -54,12 +55,19 @@
     <div class="flex flex-col gap-4">
       {#if forceDelete}
         <p>
-          <b>{user.name}</b>'s account and assets will be queued for permanent deletion <b>immediately</b>.
+          <FormatMessage message={$json('admin.user_delete_immediately')} values={{ user: user.name }} let:message>
+            <b>{message}</b>
```

Comment:
So now the entire message will be bold? Is there a way to keep the existing formatting?

Reply:
The slot is only used for message parts that have a tag. In this case it will be used for `<b>{user}</b>` and `<b>immediately</b>`. Only those parts will be bold.

Reply:
Oh got it, thanks for the explanation! :)